lib/Conversion/TorchOnnxToTorch/DefaultDomainGtoP.cpp

```diff
@@ -1665,12 +1665,12 @@ void mlir::torch::onnx_c::populateDefaultDomainGtoP(
         Value off = rewriter.create<Torch::AtenSelectIntOp>(loc, selectTy,
                                                             values, zero, zero);
         off = rewriter.create<Torch::AtenItemOp>(
-            loc, rewriter.getType<Torch::IntType>(), off);
+            loc, rewriter.getType<Torch::FloatType>(), off);
```

Comment:
are we sure that it'll always be a float type?

Reply:
Now it handles both int and float types